code/game/machinery/cryo.dm

```diff
@@ -81,12 +81,26 @@
 	open_machine()
 
 /obj/machinery/atmospherics/unary/cryo_cell/container_resist()
-	if(stat & DEAD)
-		return
-	sleep(usr.stat * 1200)
 	open_machine()
 	return
 
+/obj/machinery/atmospherics/unary/cryo_cell/verb/move_eject()

```

Comment:
Why object veeeerbs
Can't you tie this behavior into something like container_resist() or relay_move() instead